File: frontend/lib/types/pending-rating.ts
export type PendingRatingItemApi = {
  event_id: string;
  meeting_id: string;
  meeting_title: string;
  event_title: string;
  place_id: string;
  place_name: string;
  place_category: string;
  scheduled_at: string;
  ended_at_calculated: string;
};

export type PendingRatingsListApi = {
  items: PendingRatingItemApi[];
};

File: backend/app/schemas/pending_rating.py
from datetime import datetime

from pydantic import BaseModel


class PendingRatingItem(BaseModel):
    event_id: str
    meeting_id: str
    meeting_title: str
    event_title: str
    place_id: str
    place_name: str
    place_category: str
    scheduled_at: datetime
    ended_at_calculated: datetime


class PendingRatingsListResponse(BaseModel):
    items: list[PendingRatingItem]
